### PR TITLE
Integrates events & users API to admin/index route

### LINKED ISSUE #668: Implement admin API for events & users in admin route
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Parent Issue #642
Work to be done:
- Add events & users model
- Implement admin API for events & users in admin route

### PR DESCRIPTION
**Pull request by geekyd**
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Implements events & users API to admin/index route
- Adds admin-statistics-event model
- Adds admin-statistics-user model

Fixes #668

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-669-open-event-frontend-fossasia.surge.sh

**Comment by CosmicCoder96** [comment]
@geekyd Please update the branch

File: tests/unit/adapters/admin-statistics-event-test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
```
Comment (harshitagupta30) [inline tests/unit/adapters/admin-statistics-event-test.js]: @geekyd why we're not using moduleFor here from /helpers/unit-helper?

**Comment by niranjan94** [comment]
Deployment link: https://pr-669-open-event-frontend-fossasia.surge.sh

File: tests/unit/adapters/admin-statistics-event-test.js (hunk 1 above)
Comment (geekyd) [inline tests/unit/adapters/admin-statistics-event-test.js]: The tests are failing if we use unit-helper, I'm checking why they fail. Thanks for pointing it :)

**Comment by geekyd** [comment]
@CosmicCoder96 @harshitagupta30 Please review. Thanks :)

**Comment by niranjan94** [comment]
Deployment link: https://pr-669-open-event-frontend-fossasia.surge.sh

File: app/routes/admin/index.js [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
+const { Route, RSVP } = Ember;
 
 export default Route.extend({
+  model() {
+    return RSVP.hash({
+      events: this.get('store').queryRecord('admin-statistics-event', {
+        filter: {
+          name : 'id',
+          op   : 'eq',
+          val  : 1
```
Comment (sumedh123) [inline app/routes/admin/index.js]: @geekyd Just wanted to know why did we use 1 here?

Comment (geekyd) [inline app/routes/admin/index.js]: @sumedh123 The data is at id 1.
Please check https://open-event-api.herokuapp.com/#admin-statistics

**Pull request merged**